src/Gui/RemoteDebugger.py

```diff
@@ -25,6 +25,7 @@
 import FreeCAD as App
 import FreeCADGui as Gui
 from PySide import QtGui
+from addonmanager_utilities import get_python_exe
```

Comment:
instead of adding a dependency to addon manager in the core Gui module I think it would be better to refactor and move this function to the core and have the addon manager (and anyone else that might need it) import it from there. Might be something for a later or separate PR, @chennes thoughts?

Reply:
I totally agree on the dependency not being correct. however if done in a later PR the debugger would no longer work in the meantime.